client/pool.c

```diff
@@ -89,6 +94,7 @@ struct connection_pool_data {
 	uint64_t nfield_out;
 	uint64_t task_index;
 	struct xdag_block *block;
+	struct wallet_list_element * wallet_address_list;
```

Comment:
I can hardly say I like that change... But ok, we can temporary do it. Until RPC for wallets is ready.

Reply:
Would say until full node is ready. 😃